src/mechanics/timeIntegration/NewmarkBase.h

```diff
@@ -29,61 +29,16 @@ class NewmarkBase : public TimeIntegrationBase
     	mMuDampingMass = rMuDampingMass;
     }
 
-    double GetDampingCoefficientMass()const
-    {
-    	return mMuDampingMass;
-    }
-
     void SetToleranceForce(double rToleranceForce)
     {
     	mToleranceForce = rToleranceForce;
     }
 
-    double GetToleranceForce()const
-    {
-    	return mToleranceForce;
-    }
-
     void SetMaxNumIterations(int rMaxNumIterations)
     {
     	mMaxNumIterations = rMaxNumIterations;
     }
 
-    int GetMaxNumIterations()const
-    {
-    	return mMaxNumIterations;
-    }
-
-    void SetNewmarkBeta(double rBeta)
```

Comment:
Why is setting/getting the parameters for beta/gamma removed? There are other schemes options possible
[see Wikipedia](https://en.wikipedia.org/wiki/Newmark-beta_method)